src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -90,10 +91,12 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
           preferences.putAll(configuration.asMap(buildCustomConfPrefix(formId)))
       );
 
-      if (forms.someSelected() && (form.getConfPanel().isValid() || forms.allSelectedFormsHaveConfiguration()))
+      if (forms.someSelected() && (form.getConfPanel().isValid() && forms.allSelectedFormsHaveConfiguration()))
```

Comment:
Good catch! Do you think it would be safe to remove the `form.getConfPanel().isValid()`? I haven't checked but now it feels like it's a redundant check because it's implicitly made in `forms.allSelectedFormsHaveConfiguration()`

Reply:
Yeah that's true.